### ISSUE TITLE
[Bug]: Zig fails to run on Enarx

### ISSUE THREAD

**Issue by rvolosatovs** [enarx/codex#15]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Current Behaviour

```
fibonacci of 7 is: 13 
 Error: Exited with i32 exit status 0
wasm backtrace:
    0:  0xd89 - <unknown>!<wasm function 7>
```

This fails only on Enarx

Enarx tracking issue https://github.com/enarx/enarx/issues/2090 (and another one to be filed)

### Expected Behaviour

Success.

### Environment Information

enarx 0.6.1

**Comment by rvolosatovs** [comment]
Did a bit more research, it seems that Wasmtime actually works fine without I/O, but Enarx fails with the same error. Looks like a missing syscall support.

**Comment by rvolosatovs** [comment]
Fixed in enarx 0.6.2
